.gitignore

```diff
@@ -26,3 +26,4 @@ dist/
 local/
 gh-pages/
 searx.egg-info/
+searx/settings_perso.yml
```

Comment:
Can you remove this change in .gitignore ?

Reply:
yup